src/codeu/chat/server/Controller.java

```diff
@@ -140,6 +177,11 @@ public ConversationHeader newConversation(Uuid id, String title, Uuid owner, Tim
       LOG.info("Conversation added: " + id);
     }
 
+    //Log should load before adding new commands.
```

Comment:
Would you just say delete the comment? It's pretty easy to interpret the meaning just based off the name of the boolean.